src/diffusers/models/unet_2d_blocks.py

```diff
@@ -1384,6 +1462,195 @@ def forward(
         return hidden_states, output_states
 
 
+class KDownBlock2D(nn.Module):
```

Comment:
great!

Reply:
Could you maybe move this just a bit in the file so that all "down blocks" stay together?